app/common/custom_exception.py

```diff
@@ -0,0 +1,49 @@
+# define Python user-defined exceptions
+class Error(Exception):
```

Comment:
Do not use `Error` name here this is actually exception.

Reply:
Made the changes as suggested.